src/fn/csv-to-docs.js

```diff
@@ -63,7 +65,7 @@ module.exports = projectDir => {
 
     .then(() => model.references.forEach(updateRef))
     .then(() => model.exclusions.forEach(removeExcludedField))
-
+    .then(() => { if(model.user) { generateCsv(model.user,projectDir + '/users.csv'); }})
```

Comment:
Use linebreaks to make this easier to read, eg:

```js
.then(() => {
  if (model.user) {
    generateCsv(model.user,projectDir + '/users.csv');
  }
})
```

If you prefer to line all the `then`s up, you can pull out the function or include the `if` block in the `generateCsv` function.